cypress/e2e/data-import/settings/unlink-from-job-profile.cy.js

```diff
@@ -0,0 +1,102 @@
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import NewFieldMappingProfile from '../../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
+import FieldMappingProfileView from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView';
+import JobProfileView from '../../../support/fragments/data_import/job_profiles/jobProfileView';
+import JobProfileEdit from '../../../support/fragments/data_import/job_profiles/jobProfileEdit';
+import ActionProfiles from '../../../support/fragments/data_import/action_profiles/actionProfiles';
+import MatchProfiles from '../../../support/fragments/data_import/match_profiles/matchProfiles';
+import NewMatchProfile from '../../../support/fragments/data_import/match_profiles/newMatchProfile';
+import { ACCEPTED_DATA_TYPE_NAMES, FOLIO_RECORD_TYPE } from '../../../support/constants';
+import Users from '../../../support/fragments/users/users';
+import ActionProfileView from '../../../support/fragments/data_import/action_profiles/actionProfileView';
+
+describe('data-import', () => {
+  describe('Settings', () => {
+    let user;
+    const actionProfile1 = {
+      typeValue: FOLIO_RECORD_TYPE.INSTANCE,
+      name: `C11116 autotest action profile1 ${getRandomPostfix()}`,
+      action: 'Create (all record types except MARC Authority or MARC Holdings)',
+    };
+    const actionProfile2 = {
+      typeValue: FOLIO_RECORD_TYPE.INSTANCE,
+      name: `C11116 autotest action profile2 ${getRandomPostfix()}`,
+      action: 'Create (all record types except MARC Authority or MARC Holdings)',
+    };
+    const actionProfile3 = {
+      typeValue: FOLIO_RECORD_TYPE.INSTANCE,
+      name: `C11116 autotest action profile3 ${getRandomPostfix()}`,
+      action: 'Create (all record types except MARC Authority or MARC Holdings)',
+    };
+    const matchProfile = {
+      profileName: `C11116 autotest match profile ${getRandomPostfix()}`,
+    };
+    const mappingProfile = {
+      name: `C11116 mapping profile ${getRandomPostfix()}`,
+    };
+    const jobProfile = {
+      profileName: `C11116 autotest job profile ${getRandomPostfix()}`,
+      acceptedType: ACCEPTED_DATA_TYPE_NAMES.MARC,
+    };
+
+    before('create test data', () => {
+      cy.createTempUser([Permissions.settingsDataImportEnabled.gui]).then((userProperties) => {
+        user = userProperties;
+        cy.login(userProperties.username, userProperties.password);
+
+        // create mapping profile
+        NewFieldMappingProfile.createMappingProfileViaApi(mappingProfile.name);
+
+        // create match profile
+        NewMatchProfile.createMatchProfileViaApi(matchProfile.profileName);
+      });
+    });
+
+    after('delete test data', () => {
+      JobProfiles.deleteJobProfile(jobProfile.profileName);
+      ActionProfiles.deleteActionProfile(actionProfile1.name);
```

Comment:
You can use array for deleting
collectionOfActionProfiles.forEach((profile) => {
        ActionProfiles.deleteActionProfile(profile.actionProfile.name);
      });

Reply:
fixed